config/prod.exs

```diff
@@ -11,11 +11,13 @@ use Mix.Config
 # before starting your production server.
 config :nudge, NudgeWeb.Endpoint,
   url: [host: System.get_env("HOSTNAME", "localhost"), port: 80],
-  cache_static_manifest: "priv/static/cache_manifest.json"
+  cache_static_manifest: "priv/static/cache_manifest.json",
+  force_ssl: [rewrite_on: [:x_forwarded_proto]]
```

Comment:
What will this do?
Do we need it to be this?

Reply:
work flow failed in format check so i changed this
used force_ssl by refering this docs https://hexdocs.pm/phoenix/using_ssl.html#force-ssl

Reply:
work flow shouldn't be failing if we haven't changed on `main`. Also fix for a bad format issue is not enabling ssl. please revert this and let's fix the format issue if any

Reply:
btw, am asking about this commit: https://github.com/code-mancers/nudge/pull/8/commits/4a566b364e5e1dbf439b55ef66ca34a024b6cc9b
You seem to have added force ssl, any reason why this change is needed?

Reply:
i thought we need to use ssl while deploying https://hexdocs.pm/phoenix/gigalixir.html#making-our-project-ready-for-gigalixir

Reply:
removed it

Reply:
oh yea, good catch. we can add it later on, for now its fine